File: java/RunnableHandler/app/src/main/java/com/egehanasal/runnablehandler/MainActivity.java
package com.egehanasal.runnablehandler;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.os.Handler;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView textView;
    TextView timeTracker;
    Button button;
    int number;
    int countTurn;
    Runnable runnable;
    Handler handler;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView = findViewById(R.id.textView);
        timeTracker = findViewById(R.id.timeTracker);
        button = findViewById(R.id.startButton);
        this.number = 0;
        this.countTurn = 0;
    }

    public void buttonPressed(View view) {
        if (this.countTurn %2 == 0) {
            handler = new Handler();
            runnable = new Runnable() {
                @Override
                public void run() {
                    textView.setText(""+number);
                    number++;
                    handler.postDelayed(runnable, 1000);
                }
            };
            handler.post(runnable);
            button.setText("Stop");
        }
        else {
            handler.removeCallbacks(runnable);
            button.setText("Start");
            timeTracker.setText("Last time: " + number);
            this.number = 0;
            textView.setText(""+number);
        }
        this.countTurn++;
    }
}

File: kotlin/FragmentKotlin/app/src/main/java/com/egehanasal/fragmentkotlin/MainActivity.kt
package com.egehanasal.fragmentkotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun firstFragment(view:View) {
        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()

        val firstFragment = BlankFragment()
        // add yazsak üstüne eklicekti. sadece tıklanmış butona ait fragment'i görmek için replace dedik
        fragmentTransaction.replace(R.id.frameLayout, firstFragment).commit()
    }

    fun secondFragment(view:View) {
        val fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager.beginTransaction()

        val secondFragment = BlankFragment2()
        fragmentTransaction.replace(R.id.frameLayout, secondFragment).commit()
    }
}